sdk/eventgrid/service.projects

```diff
@@ -1,5 +1,34 @@
+<!--
+  This file allows overrides and extensions to the build configuration defined
+  by the central engineering system, and is considered during pipeline builds for
+  CI, test runs, and package release.
+
+  Based on the SDKType variable (msbuild property or env variable) this will filter
+  the set of projects to to match what type of SDK we are interested in building.
+
+  SDKType values
+   - "all" will build all projects
+   - "client" will build the new track 2 client/data libraries
+   - "mgmtclient" will build the track 2 mgmt libraries
+   - "data" will build the track 1 data plane libraries
+-->
+
 <Project>
-  <ItemGroup>
-    <ProjectReference Remove="$(MSBuildThisFileDirectory)Microsoft.Azure.*\**\*.csproj" Condition="'$(AZURE_EVENTGRID_TEST_MODE)' == 'Live'" />
```

Comment:
It looks like we were excluding the track 1 library tests during live mode in the past so I assume we probably want to exclude them still.

Reply:
This is for the old pipeline which exclude Track 1, now the pipeline is using SDKType to define the scope for testing.

Reply:
Looks like this is causing issues in the live test pipeline - I think we need to add this line back.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1946344&view=results